pkg/rpcclient/actor/waiter.go

```diff
@@ -319,13 +319,17 @@ func (w *EventWaiter) WaitAny(ctx context.Context, vub uint32, hashes ...util.Ui
 	}
 
 	select {
-	case _, ok := <-bRcvr:
+	case b, ok := <-bRcvr:
 		if !ok {
 			// We're toast, retry with non-ws client.
 			wsWaitErr = ErrMissedEvent
 			return
 		}
-		waitErr = ErrTxNotAccepted
+		// We can easily end up in a situation when subscription was performed too late and
+		// the desired transaction and VUB-th block have already got accepted before the
+		// subscription happened. Thus, always retry with non-ws client, it will perform
+		// AER requests and make sure.
```

Comment:
Subscribe first, send afterwards? We need some issue for it.

Reply:
See https://github.com/nspcc-dev/neo-go/issues/2805.